presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergSessionProperties.java

```diff
@@ -240,6 +244,16 @@ public IcebergSessionProperties(
                         CACHE_ENABLED,
                         "Enable cache for Iceberg",
                         cacheConfig.isCachingEnabled(),
+                        false),
+                stringProperty(
+                        NESSIE_REFERENCE_NAME,
+                        "Nessie reference name to use",
+                        nessieConfig.getDefaultReferenceName(),
+                        false),
+                stringProperty(
+                        NESSIE_REFERENCE_HASH,
+                        "Nessie reference hash to use",
```

Comment:
Why don't we have config for NESSIE_REFERENCE_HASH?

Reply:
because this is generally something a user would only specify at runtime when actually reading data at a particular hash. Having a config option for that would just be confusing to users because they would have to know the hash upfront.

Reply:
So does it mean usually different users would use different hash and they don't usually share? and thus we don't have any need to set a default value on a cluster level